### DISCUSSION TITLE
Basic timer function not working

### DISCUSSION THREAD

**Discussion by cer2225** [arendst/Tasmota#17217] [Q&A]
Sorry if this is a basic question but I have just flashed my Sonoff Basic with Tasmota and all I need it to do is turn lights on and off at set times.  I have added DST and set the Time zone to 99.  When I look at the output when I type Status 7, I do see my correct local time.

Other than flashing the device, renaming it, setting up DST and putting it on my network, I have done no other configuration. I have enabled timers, armed them, and set timer 1 to turn on output 1 at 6AM and timer 2 to turn off output 1 at 11 pm.  

I am not sure what I am missing but the timers do not work.  I am sure it is something simple so any help would be appreciated.

Thank You

**Comment by sfromis** [comment]
You did enable all the needed places for a repeating timer?
![image](https://user-images.githubusercontent.com/47082390/204566366-485a5cf5-2b8d-4230-a7a9-6883572b6e19.png)

**Comment by cer2225** [comment]
Thanks for the reply.
I do have all of those selected. (using time based and not sun)
A friend suggest that I select the Sonoff module type (Under Configuration) which I had not done.  Maybe that's what I am missing?

**Reply by sfromis** [reply]
Well, default module type if doing nothing else is already "Sonoff Basic", not sure what the change suggestion even was? Anyway, module/template config is important to it working for relay/button actions, not to timers working. 
The command Timers will dump the whole timers config, can be useful for verification.....

**Reply by cer2225** [reply]
Thanks.  When I get home from work tonight, I will spend some more time on it and if needed I will get the output from the timers command and paste it here.

I appreciate the help.

Thanks

**Reply by luciangpl** [reply]
Check the timezone, for me, that was the cause
